posthog/hogql/database/schema/short_links.py

```diff
@@ -0,0 +1,28 @@
+from posthog.hogql.database.models import (
+    StringDatabaseField,
+    DateTimeDatabaseField,
+    IntegerDatabaseField,
+    Table,
+    FieldOrTable,
+)
+
+
+class ShortLinksTable(Table):
```

Comment:
This is cool. Are we automatically adding a `team_id` `WHERE` clause to filter to each individual team's links?